google/cloud/bigtable/internal/data_connection_impl_test.cc

```diff
@@ -194,10 +196,12 @@ Options CallOptions() {
 }
 
 std::shared_ptr<DataConnectionImpl> TestConnection(
-    std::shared_ptr<BigtableStub> mock) {
+    std::shared_ptr<BigtableStub> stub,
+    std::shared_ptr<MutateRowsLimiter> limiter =
+        std::make_shared<NoopMutateRowsLimiter>()) {
```

Comment:
Avoid defaulting parameters when you control all the callers?

Reply:
In this case, I personally prefer to have less clutter than to be more explicit / deliberate.

49/50 times we want the default. 1/50 times we don't.